admin/write-news.php

```diff
@@ -0,0 +1,113 @@
+<?php
+    session_start();
+    if(!$_SESSION["nama_admin"]){
```

Comment:
Codacy found an issue: [Direct use of $_SESSION Superglobal detected.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)